commons-numbers-examples/examples-jmh/src/main/java/org/apache/commons/numbers/examples/jmh/DoubleUtils.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.numbers.examples.jmh;
+
+import org.apache.commons.rng.UniformRandomProvider;
+
+/** Class containing utility methods for working with doubles.
+ */
+public final class DoubleUtils {
```

Comment:
Does all this have to be public? I think it can move to the arrays package and be package private.

Reply:
Yep. Done.